doc/modules/isotonic.rst

```diff
@@ -7,12 +7,12 @@ Isotonic regression
 .. currentmodule:: sklearn.isotonic
 
 The class :class:`IsotonicRegression` fits a non-decreasing real function to
-1-dimensional data. It solves the following problem:
+1-dimensional data. It solves the following problem: minimize
 
-  minimize :math:`\sum_i w_i (y_i - \hat{y}_i)^2`
-
-  subject to :math:`\hat{y}_i \le \hat{y}_j` whenever :math:`X_i \le X_j`,
+.. math::
+    \sum_i w_i (y_i - \hat{y}_i)^2
```

Comment:
I assume that we should use `\argmin` here instead of the "minimize"

Reply:
Do you mean to just put `\operatorname{argmin}`? If I'm not mistaken this is a QP in all `\hat{y}_i` so it might be a bit difficult to write "with respect to something" (i.e., the subscript of argmin), which I think is the reason why the original author vaguely wrote "minimize".